tests/behave/features/search_legal_advisers.feature

```diff
@@ -0,0 +1,9 @@
+Feature: Search for Education and Discrimination categories
+
+  Scenario Outline: Find a legal adviser in <category>
+    Given I am a on the fala landing page
+    When I search for a legal adviser with postcode "<postcode>" and category "<category>"
+    Then I should see a list of legal advisers
+    Examples: Search
+        | postcode         | category |
+        | W1A | disc        |
```

Comment:
Most editors can auto-format `.feature` files to align these table brackets. It's usually worth doing as it avoids subsequent diffs changing unrelated lines.